src/components/Comments.jsx

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { useEffect } from "react";
+import { useState } from "react";
+
+const Comments = ({review_id}) => {
+    
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        setIsLoading(true);
+        axios.get(`https://nc-games-no2.onrender.com/api/reviews/${review_id}/comments`)
+        .then(({data}) => {
+            setComments(data)
+            setIsLoading(false)
+        })
+    }, []);
+
+    if (isLoading) {
+        return <p>Loading comments...</p>
+      }
+
+    return (
+        <>
+        <h2>Comments</h2>
+        <ul>
+            {comments.map((comment) => {
+             return (
+              <li key={comment.comment_id} className="comment">
+                <p>{comment.body}</p>
+                <p>Written by: {comment.author}</p>
+                <time>Created at: {comment.created_at}</time>
+                <p>Votes: {comment.votes}</p>
+              </li>
+              )
+            })}
+        </ul>
+        </>
+    )
+}
+
+export default Comments
```

Comment:
Nice use of loading state here to keep the user informed. You could export the functionality for fetching here to have all axios requests in one place, however this is personal preference.